### DISCUSSION TITLE
Auto create alias rules (regex help needed)

### DISCUSSION THREAD

**Discussion by SwarfegaGit** [simple-login/app#1929] [Q&A]
I have no idea why, but I just can't seem to ever get my head around regex.
I have a custom domain where all my aliases are in the format

website.*****@example.com

The ***** are 5 random letters or numbers.  I want to replicate this so that I can create aliases on the fly without having to log into SimpleLogin.  Can anyone tell me the correct regex syntax to replicate this?

*Edit*

After lots of playing, I have this. It looks like it works. Can someone confirm, please?

[a-zA-Z]+.\w{5}

**Comment by springcomp** [comment]
@SwarfegaGit a correct regex would be:

`website\.[0-9A-Za-z]{5}`
